src/data/connectors/index.ts

```diff
@@ -14,14 +14,23 @@ export * from "./measurement";
 export const Measurement = Mongoose.model<IMeasurementModel>("Measurement", MeasurementSchema);
 export const User = Mongoose.model<IUserModel>("User", UserSchema);
 
-const user = process.env.ROM_DB_USER;
-const pw = process.env.ROM_DB_PW;
+let ROM_DB_URL;
 
-export const MongooseConnection = Mongoose.connect("mongodb://" + user + ":" + pw + "@ds155737.mlab.com:55737/romdb")
+if (process.env.NODE_ENV === "prod") {
+  ROM_DB_URL = process.env.ROM_DB_PROD;
+} else if (process.env.NODE_ENV === "test") {
+  ROM_DB_URL = process.env.ROM_DB_TEST;
+}
+
+export const MongooseConnection = Mongoose.connect(ROM_DB_URL)
   .catch((connectError) => {
     console.error("Could not connect to MongoDB on mlab", connectError);
   })
-  .then(() => seeder.seed(seedData, { dropCollections: false }))
+  .then(() => {
+    if (process.env.NODE_ENV === "test" && process.env.NODE_ENV !== "prod") {
```

Comment:
The last part is not necessary. The variable can never be equal to "test" and still be "prod"